lldb/source/Plugins/ABI/LoongArch/ABISysV_loongarch.cpp

```diff
@@ -644,32 +644,22 @@ void ABISysV_loongarch::AugmentRegisterInfo(
     std::vector<lldb_private::DynamicRegisterInfo::Register> &regs) {
   lldb_private::RegInfoBasedABI::AugmentRegisterInfo(regs);
 
+  static const std::unordered_map<std::string, std::string> reg_aliases = {
```

Comment:
There is `llvm/include/llvm/ADT/StringMap.h` for these sorts of purposes. We only do this once on process start but given it's the more "specific" type to LLVM you might as well use it.